.dockerignore

```diff
@@ -4,4 +4,5 @@
 !/go.mod
 !/go.sum
 !/.golangci.yml
-!/vendor
\ No newline at end of file
+!/vendor
+!/docs
```

Comment:
Why? This would make the validation failed

Reply:
That's an exception so we're good 👍  (ignore everything but /docs)

Reply:
Oh indeed my bad!